File: Signiture-Backend/delivery/api/src/main/java/pcy/study/api/domain/token/helper/TokenHelper.java
package pcy.study.api.domain.token.helper;

import pcy.study.api.domain.token.model.Token;

import java.util.Map;

public interface TokenHelper {

    Token issueAccessToken(Map<String, Object> data);
    Token issueRefreshToken(Map<String, Object> data);

    Map<String, Object> validationTokenWithThrow(String token);
}

File: Signiture-Backend/kotlin/delivery/account/src/main/kotlin/pcy/study/account/domain/token/helper/TokenHelper.kt
package pcy.study.account.domain.token.helper

import pcy.study.account.domain.token.model.Token

interface TokenHelper {

    fun issueAccessToken(data: Map<String, Any>): Token
    fun issueRefreshToken(data: Map<String, Any>): Token

    fun validationTokenWithThrow(token: String): Map<String, Any>
}